Assets/Plugins/CountlySDK/Services/SessionCountlyService.cs

```diff
@@ -96,17 +98,45 @@ public async Task ExecuteBeginSessionAsync()
 			//if (ConsentModel.CheckConsent(FeaturesEnum.Sessions.ToString()))
 			//{
 			var requestParams =
-				new Dictionary<string, object>
+				new Dictionary<string, object>();
+
+
```

Comment:
Begin_session requst doesn't send location like according to the spec it should.